packages/experiments/src/components/Button/MenuButton/MenuButton.state.ts

```diff
@@ -31,6 +31,7 @@ export class MenuButtonState extends BaseState<IMenuButtonProps, IMenuButtonView
     if (!disabled) {
       if (onClick) {
         onClick(ev);
+        ev.stopPropagation();
```

Comment:
This seems wrong.

Why are you stopping propogation only if onClick is passed in?

Reply:
I think stopping propogation might be OK if we are doing work for the click and don't expect other clicks to be handled at a higher layer. It just seems wrong to do it inside of the if check.

Reply:
We do want it to stop propagation only when the `MenuButton` is not `disabled` right?